app/views/home/api_v1.html.erb

```diff
@@ -196,6 +196,48 @@
   <div class="col-md-12">
     <table class="table table-bordered api">
       <tr>
+        <td class="GET def">GET</td>
+        <td class="method">Key</td>
+        <td class="code">/api/v1/projects/XX/key/</td>
+        <td class="args">contribution_key(req)</td>
+      </tr>
+      <tr class="more" style="">
+        <td colspan="4">
+          <b>Response Codes</b>
+          <ul>
+            <li>Success: 302 Found</li>
+          </ul>
+          <ul>
+            <li>Fail: 404 Not Found</li>
+          </ul>
+           <b>Data</b>
+          <pre>
+  {
```

Comment:
Are you referring to this?
{
    "contribution_key" => "contribution key (string)"
}

If I indent this it will be awkwardly indented on the website.